src/java/org/apache/cassandra/schema/TableMetadata.java

```diff
@@ -196,6 +197,8 @@ public enum Kind
     public final DataResource resource;
     public TableMetadataRef ref;
 
+    protected Set<CqlConstraint> constraints;
```

Comment:
@bbotella btw there is TableParams class in this package, you need to update it all with constraints stuff and bump version in NodeVersion and Version to V5.

Reply:
Do I really need to update TableParams? The constraints come at TableMetadata level (so, same level as TableParams). I think I only need to make sure that TableMetadata is serialized/deserialized properly right?